frontend/src/lib/components/chart/chart-page/encoding/chart-encoding/encoding-components/ModelsEncodingMultiChoice.svelte

```diff
@@ -1,52 +1,33 @@
 <script lang="ts">
 	import { models } from '$lib/stores';
-	import { svelecteRenderer } from '$lib/util/util';
 	import Checkbox from '@smui/checkbox';
-	import Svelecte from 'svelecte';
 	import { createEventDispatcher } from 'svelte';
-	import { dndzone } from 'svelte-dnd-action';
-	import EncodingContainer from './EncodingContainer.svelte';
+	import MultiSelect from 'svelte-multiselect';
 
 	export let stringValues: string[];
 
-	const dispatch = createEventDispatcher<{ selected: string[] }>();
-
-	let options: { value: string; label: string }[] = [];
-	let value: string[] = [];
+	let value = stringValues;
 
-	// initial options & values
-	$models.forEach((m) => {
-		options.push({ value: m, label: m });
-	});
-	value = stringValues;
-
-	function updateDragOrder(val: string[]) {
-		// check if all elements are numbers (dndzone's place holder)
-		dispatch('selected', val);
-	}
+	const dispatch = createEventDispatcher<{ selected: string[] }>();
 
-	$: updateDragOrder(value);
+	$: dispatch('selected', value);
 </script>
 
-<EncodingContainer>
-	<div class="flex flex-col">
-		{#if value[0] != ''}
-			<Svelecte
-				style="width: 280px; flex:none;"
-				bind:value
-				{options}
-				{dndzone}
-				multiple
-				renderer={svelecteRenderer}
-				placeholder="Select Systems..."
-			/>
-		{/if}
-		<div class="ml-auto flex items-center">
-			<span>All Systems</span>
-			<Checkbox
-				checked={value[0] == ''}
-				on:click={() => (value[0] === '' ? (value = []) : (value = ['']))}
-			/>
-		</div>
+<div class="flex w-full flex-col">
+	{#if value.length === 0 || value[0] != ''}
+		<MultiSelect
+			bind:selected={value}
+			options={$models}
+			liSelectedClass="!bg-primary-light ![&>svg]:fill-primary"
+			outerDivClass="!w-full !border-grey-light !py-1 !bg-white"
+			liActiveOptionClass="!bg-primary-light"
+		/>
+	{/if}
+	<div class="ml-auto flex items-center">
+		<span>All Systems</span>
+		<Checkbox
+			checked={value.length > 0 && value[0] == ''}
```

Comment:
I didn't do that either but maybe we should add a comment somewhere do explain this empty string?